addons/web/static/src/actions/action_service.js

```diff
@@ -53,7 +53,7 @@ function makeActionManager(env) {
   // The state action (or default user action if none) is loaded as soon as possible
   // so that the next "doAction" will have its action ready when needed.
   const actionParams = _getActionParams(_getHashState());
-  if (actionParams) {
+  if (actionParams && Number.isInteger(actionParams.actionRequest)) {
```

Comment:
Agree with that solution, however `actionRequest` has already been parsed by `_getHashState` if it's a number so maybe use `typeof actionParams.actionRequest === "number"` (for consistency towards `_loadAction` type checking)?

Btw all number values from the hash state are currently parsed as numbers (floats as well): I don't know if this could cause more issues than just parsing the integers, since these values would not match anything anyway. Any thoughts?

Reply:
cids and comma separated integers (e.g. `cids=2,3`) I think. I hope it is correctly parsed. I agree with your suggestion though.

Reply:
`active_ids` is parsed as a comma-separated list of numbers (`cids` is not used by the action service) and this is already tested. I think we're safe from type errors, but maybe it's weird to potentially send *float* action ids... No strong opinion on this though, just pointing it out